app/controllers/helper/ShapefilesCreatorHelper.java

```diff
@@ -0,0 +1,443 @@
+package controllers.helper;
+
+import java.io.*;
+import java.util.*;
+import java.util.zip.*;
+import scala.Option;
+import scala.runtime.AbstractFunction0;
+
+import org.geotools.data.DataUtilities;
+import org.geotools.data.DefaultTransaction;
+import org.geotools.data.Transaction;
+import org.geotools.data.collection.ListFeatureCollection;
+import org.geotools.data.shapefile.ShapefileDataStore;
+import org.geotools.data.shapefile.ShapefileDataStoreFactory;
+import org.geotools.data.simple.SimpleFeatureCollection;
+import org.geotools.data.simple.SimpleFeatureSource;
+import org.geotools.data.simple.SimpleFeatureStore;
+import org.geotools.feature.simple.SimpleFeatureBuilder;
+import org.geotools.geometry.jts.JTSFactoryFinder;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.locationtech.jts.geom.Coordinate;
+
+import models.attribute.GlobalAttributeForAPI;
+import models.attribute.GlobalAttributeWithLabelForAPI;
+import controllers.NeighborhoodAttributeSignificance;
+import controllers.StreetAttributeSignificance;
+
+
+
+/**
+ * This example reads data for point locations and associated attributes from a comma separated text
+ * (CSV) file and exports them as a new shapefile. It illustrates how to build a feature type.
```

Comment:
I'm assuming that this comment is no longer totally accurate?

Reply:
yup